src/mlpack/bindings/cli/cli_option.hpp

```diff
@@ -131,59 +131,59 @@ class CLIOption
       #undef BASH_RED
       #undef BASH_CLEAR
 
-      outstr << "Parameter --" << boostName << " (" << data.alias << ") "
+      outstr << "Parameter --" << cliName << " (" << data.alias << ") "
              << "is defined multiple times with the same identifiers."
              << std::endl;
     }
 
-    CLI::Add(std::move(data));
+    CMD::Add(std::move(data));
 
     // Set some function pointers that we need.
-    CLI::GetSingleton().functionMap[tname]["DefaultParam"] =
+    CMD::GetSingleton().functionMap[tname]["DefaultParam"] =
         &DefaultParam<N>;
-    CLI::GetSingleton().functionMap[tname]["OutputParam"] =
+    CMD::GetSingleton().functionMap[tname]["OutputParam"] =
         &OutputParam<N>;
-    CLI::GetSingleton().functionMap[tname]["GetPrintableParam"] =
+    CMD::GetSingleton().functionMap[tname]["GetPrintableParam"] =
         &GetPrintableParam<N>;
-    CLI::GetSingleton().functionMap[tname]["StringTypeParam"] =
+    CMD::GetSingleton().functionMap[tname]["StringTypeParam"] =
         &StringTypeParam<N>;
-    CLI::GetSingleton().functionMap[tname]["GetParam"] = &GetParam<N>;
-    CLI::GetSingleton().functionMap[tname]["GetRawParam"] = &GetRawParam<N>;
-    CLI::GetSingleton().functionMap[tname]["AddToPO"] = &AddToPO<N>;
-    CLI::GetSingleton().functionMap[tname]["MapParameterName"] =
+    CMD::GetSingleton().functionMap[tname]["GetParam"] = &GetParam<N>;
+    CMD::GetSingleton().functionMap[tname]["GetRawParam"] = &GetRawParam<N>;
+    CMD::GetSingleton().functionMap[tname]["AddToPO"] = &AddToPO<N>;
+    CMD::GetSingleton().functionMap[tname]["MapParameterName"] =
         &MapParameterName<N>;
-    CLI::GetSingleton().functionMap[tname]["SetParam"] = &SetParam<N>;
-    CLI::GetSingleton().functionMap[tname]["GetPrintableParamName"] =
+    CMD::GetSingleton().functionMap[tname]["SetParam"] = &SetParam<N>;
```

Comment:
I think `SetParam` was used only during the process of parsing with `boost::program_options`, and the lambdas that you are now passing to CLI11 probably mean that `SetParam` isn't used anymore.  Do you think you could check to see if there are any other uses, and if not, maybe you could remove that function entirely?

Reply:
Actually, I thought about it, but I was afraid to remove a lot of stuff before making it working.

Reply:
It is still used when generating python code in `print_input_processing.hpp`
I am not sure why it is used in Python and Julia, but it is still used there.